.gitignore

```diff
@@ -15,6 +15,9 @@
 # emacs stuff
 *~
 
+# Ignore the bazel remote cache key file
```

Comment:
End comments with `.` (and below).

Reply:
None of the comments in .gitignore have a . :(

Reply:
Done.